tasks/Darwin.yml

```diff
@@ -11,7 +11,7 @@
 - name: Symlink OpenJDK to system-wide Java VMs
   ansible.builtin.file:
     state: link
-    src: /usr/local/opt/openjdk@17/libexec/openjdk.jdk
+    src: "{{ homebrew_prefix }}/opt/openjdk@17/libexec/openjdk.jdk"
```

Comment:
Can you please add a default `homebrew_prefix` variable in `defaults/main.yml` and set it to `/usr/local` so that this can be changed but the default doesn't break our existing setup.